client/concordclient/src/concord_client.cpp

```diff
@@ -117,6 +134,18 @@ void ConcordClient::send(const bft::client::WriteConfig& config,
   client_pool_->SendRequest(config, std::forward<bft::client::Msg>(msg), callback);
 }
 
+void ConcordClient::CheckAndReConnectGrpcConnections() {
```

Comment:
Lower-case function names.

Reply:
Thanks, @robem , I will do this in my next commit.